lib/WP_Auth0_Api_Client.php

```diff
@@ -619,6 +630,14 @@ public static function update_connection( $domain, $app_token, $id, $payload ) {
 		$headers['Authorization'] = "Bearer $app_token";
 		$headers['content-type']  = 'application/json';
 
+		unset( $payload->name );
+		unset( $payload->strategy );
+		unset( $payload->id );
+
+		if ( ! empty( $payload->enabled_clients ) ) {
```

Comment:
If the array is not consecutive then `json_encode` converts this to an object rather than an array.

Reply:
If `$payload->enabled_clients` is anything other than an array with consecutive keys (like, if one of the keys in the middle is removed) then `json_encode()` turns it into an `object` which is rejected by the server.